{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "description": null,
    "dev_url": null,
    "doc_source_url": null,
    "doc_url": null,
    "home": "https://bitbucket.org/pytest-dev/pytest-timeout",
    "icon_hash": null,
    "icon_url": null,
    "identifiers": null,
    "keywords": null,
    "license": "MIT",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "recipe_origin": null,
    "run_exports": {},
    "source_git_url": null,
    "source_url": "https://pypi.io/packages/source/p/pytest-timeout/pytest-timeout-1.2.0.tar.gz",
    "subdirs": [
      "linux-64"
    ],
    "summary": "This is a plugin which will terminate tests after a certain timeout.",
    "tags": null,
    "text_prefix": false,
    "timestamp": 0,
    "version": "1.2.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "cf202003",
    "main",
    "cf201901"
  ],
  "package": "pytest-timeout-1.2.0-py34_0.tar.bz2",
  "repodata": {
    "build": "py34_0",
    "build_number": 0,
    "depends": [
      "pytest >=2.8.0",
      "python 3.4*"
    ],
    "license": "MIT",
    "md5": "39c011db6f481096544bd09e0908c3d7",
    "name": "pytest-timeout",
    "sha256": "6f6263802c4c639c838d3ddec319978805274da39eda414430595ab80af86c43",
    "size": 11783,
    "subdir": "linux-64",
    "version": "1.2.0"
  },
  "repodata_version": 1,
  "subdir": "linux-64",
  "url": "https://conda.anaconda.org/conda-forge/linux-64/pytest-timeout-1.2.0-py34_0.tar.bz2"
}